### PR TITLE
Ack addtnl form of interop that's important: with legacy tech

### PR DESCRIPTION
**Pull request by dhh1128**
Signed-off-by: Daniel Hardman <daniel.hardman@gmail.com>

This replaces a comment I left in the google doc version of the spec, before I noticed @talltree 's comment that it was frozen and the content should be managed here in Github.

BTW, I have some open comments from section 6 in that same document. There is no comment from Drummond indicating that that section is locked. But I can submit a PR with those comments, if this is the canonical place for that section as well. What do you advise?

### CODE REVIEW

**Comment by andorsk** [comment]
This needs to be rebased with main. Was there a reason this hasn't been merged yet? @wenjing would you like to review this ( it was made in August ) or should someone else do so ( I'm happy to help review it ) ? @dhh1128 mind resolving any conflicts so we can get this into main?

**Comment by dhh1128** [comment]
Merge conflicts resolved.

**Comment by andorsk** [comment]
Thanks @dhh1128 !

**Review by andorsk — APPROVED** [review]
This LGTM! I'd like one a view on it from one other person though before we merge. Will add a few more reviewers.

**Review by talltree — APPROVED** [review]
LGTM

**Review by wenjing — APPROVED** [review]

**Comment by andorsk** [comment]
Merged based on TechArch discussion Nov 3.

File: spec.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Whether from the perspective of an implementer, a customer, or a policymaker, th
 
 - **Reliability.** A well-defined architecture enables the development of software components and applications that can be trusted to act in predictable, reliable ways—and that can expect other components and applications to do the same.
 
-- **Interoperability and vendor independence.** As with the TCP/IP stack, the Bluetooth stack, the NFC stack, or other protocol stacks, implementations from multiple vendors can and should be interoperable, both with current standards, and as they evolve over time. Customers should be able to switch between them without major loss of functionality.
+- **Interoperability and vendor independence.** As with the TCP/IP stack, the Bluetooth stack, the NFC stack, or other protocol stacks, implementations from multiple vendors can and should be interoperable, and customers should be able to switch between them without major loss of functionality. In addition, we want the ToIP stack to be as interoperable as is practical with existing internet technologies that may not embody ToIP goals, as long as the guiding principles of Trust Over IP are not frustrated.
```
Comment (talltree) [inline spec.md:53]: ...between them while maintaining baseline functionality.

Whenever practical, ToIP will leverage existing technologies that support and promote the ToIP principles.

Comment (talltree) [reply spec.md:53]: This suggestion has been addressed by PR #59.

**Pull request merged**